build.sbt

```diff
@@ -18,16 +18,11 @@ licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 // publish to the Sonatype repository
 publishTo := sonatypePublishTo.value
 
-// retrieve secrets to sign files
-pgpPublicRing := file("ci/pubring.asc")
-pgpSecretRing := file("ci/secring.asc")
-pgpPassphrase := sys.env.get("PGP_PASSPHRASE").map(_.toArray)
```

Comment:
We should keep these lines, or provide a replacement.

Reply:
IIUC these are settings for `sbt-pgp` so we can keep them even with the switch to `sbt-ci-release` as it uses `sbt-pgp`. I could revert this change